dotNETStandard/HealthVault.Mobile/cs/Xamarin.Core/ViewModels/MedicationsMainViewModel.cs

```diff
@@ -98,7 +112,7 @@ private void UpdateDisplay(IEnumerable<Medication> items)
         {
             var medicationsMainPage = new MedicationsSummaryPage()
             {
-                BindingContext = new MedicationsSummaryViewModel(medication, this.ingredientChoices, this.thingClient, this.recordId, this.NavigationService),
+                BindingContext = new MedicationsSummaryViewModel(medication, this.connection, this.NavigationService),
```

Comment:
there is no IOC in samples ? 

---
In reply to: [117592508](https://github.com/Microsoft/healthvault-samples/pull/11#discussion_r117592508) [](ancestors = 117592508)